mk/containerized.mk

```diff
@@ -19,7 +19,12 @@ pull: ## Download SVC images
 build:  ## Build SVC images
 build: DOCKER_COMPOSE_ENV = DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1
 
-push: ## Push SVC container images to a registry. Requires previous "docker login"
+GIT_LATEST_TAG != git describe --tags --abbrev=0
+push: ## Push nvmeof and nvmeof-cli containers images to quay.io registries
```

Comment:
You can achieve the same behaviour by properly setting the variables, without overriding the `push` target recipe:
```make
registry-login: CONTAINER_REGISTRY_USER ?= ## Container registry user
registry-login: CONTAINER_REGISTRY_PASSWORD ?= ## Container registry password
registry-login: ## Login to container registry for pushing images
    docker login -u $(CONTAINER_REGISTRY_USER) -p $(CONTAINER_REGISTRY_PASSWORD) $(CONTAINER_REGISTRY)

push: registry-login
push: NVMEOF_VERSION = $(GIT_LATEST_TAG)
push: NVMEOF_CLI_VERSION = $(GIT_LATEST_TAG)
push: SVC = nvmeof nvmeof-cli
```

Reply:
I think the current way could be left as it is now, without mentioning setting variables and most importantly displaying which image we are pushing and to which registry we are pushing to in a quick glance.